### PR TITLE
docs : FAQ in in user guide

### PR DESCRIPTION
**Pull request by KishaniKandasamy**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [X] Bugfix

**Related issues**

fix: #1804

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -70,3 +70,6 @@ $ swagger generate spec -o ./docs/en/latest/api/api.yaml --scan-models
 ```shell
 $ swagger-markdown -i ./docs/en/latest/api/api.yaml
 ```
+### 6. What is the default strategy when import a duplicate route?
```
Comment (liuxiran) [inline docs/en/latest/FAQ.md:73]:
```suggestion

### 6. What is the default strategy when import a duplicate route?
```

Comment (juzhiyuan) [reply docs/en/latest/FAQ.md:73]: image

Seems no change?

Comment (KishaniKandasamy) [reply docs/en/latest/FAQ.md:74]: may be for leaving a line🤔?

Comment (liuxiran) [reply docs/en/latest/FAQ.md:74]: yep, only to fix the lint error

Comment (KishaniKandasamy) [reply docs/en/latest/FAQ.md:74]: 👍👍

**Review by liuxiran — APPROVED** [review]
It seems that there are some thing wrong with CI, maybe @KishaniKandasamy you have to push hard to retry

**Review by starsz — APPROVED** [review]

**Pull request closed without merging**